ci/sandbox/release-pipeline.yaml

```diff
@@ -106,11 +108,43 @@ spec:
       - get: src
         trigger: true
 
+      - task: build-vmc
+        privileged: true
+        config: &build_config
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: vito/oci-build-task
+            version:
+              digest: sha256:cfb2983956145f54a4996c2aff5fc598856c8722922a6e73f9ebfa3d9b3f9813
+          params:
+            CONTEXT: src
+          inputs:
+          - name: src
+          outputs:
+          - name: image
```

Comment:
While it's in our mind do we want to use per-image volumes to avoid the future gotcha?